### ISSUE TITLE
NullPointerException when Decoding values */* XMLs

### ISSUE THREAD

**Issue by haihack** [iBotPeaches/Apktool#1399]
### Information
1. **Apktool Version (`apktool -version`)** - 2.2.1
2. **Operating System (Mac, Linux, Windows)** - Windows 7
3. **APK From? (Playstore, ROM, Other)** - Playstore

### Stacktrace/Logcat
```
I: Using Apktool 2.2.1 on myapp.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: C:\Users\Administrator\AppData\Local\apktool\framework\1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
Exception in thread "main" java.lang.NullPointerException
        at brut.androlib.res.data.value.ResStyleValue.serializeToResValuesXml(ResStyleValue.java:58)
        at brut.androlib.res.AndrolibResources.generateValuesFile(AndrolibResources.java:516)
        at brut.androlib.res.AndrolibResources.decode(AndrolibResources.java:267)
        at brut.androlib.Androlib.decodeResourcesFull(Androlib.java:132)
        at brut.androlib.ApkDecoder.decode(ApkDecoder.java:108)
        at brut.apktool.Main.cmdDecode(Main.java:166)
        at brut.apktool.Main.main(Main.java:81)
```

### Steps to Reproduce
1. `apktool d myapp.apk `

### APK
If this APK can be freely shared, please upload/attach a link to it.
https://drive.google.com/file/d/0B8cF3QC8ByHORTNZU19CQXlXQ0k/view?usp=sharing

**Comment by iBotPeaches** [comment]
Fixed in the upcoming 2.2.2 release.

```
➜  Bug1399 apktool d myapp.apk 
I: Using Apktool 2.2.2-b6127d-SNAPSHOT on myapp.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
➜  Bug1399 
```

**Comment by sweatyc** [comment]
Confirm it's fixed in 2.3.3

**Comment by ZUFaulfuerUsername** [comment]
I've got 2.3.4 and I still get this error
